packages/core-components/src/components/DependencyGraph/types.ts

```diff
@@ -89,6 +90,39 @@ export namespace DependencyGraphTypes {
     props: RenderNodeProps<T>,
   ) => ReactNode;
 
+  /**
+   * Properties of {@link DependencyGraphTypes.RenderEdgeFunction} for {@link DependencyGraphTypes.DependencyEdge}
+   *
+   * @public
+   */
+  export type RenderEdgeProps<T = unknown> = {
+    edge: DependencyEdge<T>;
+    id: dagre.Edge;
+  };
+
+  /**
+   * Custom React component for graph {@link DependencyGraphTypes.DependencyEdge}
+   *
+   * @public
+   */
+  export type RenderEdgeFunction<T = {}> = (props: {
+    edge: T & {
```

Comment:
Not sure what happened here - this was all meant to be part of `RenderEdgeProps` wasn't it? The other callbacks don't have this extra complexity in them.